data/categories.json

```diff
@@ -60,5 +60,68 @@
     "labels": ["T-compiler"],
     "links": [],
     "tags": ["compiler", "lang-rust"]
+},
+{
+    "id": "compiler-errors",
+    "title": "compiler (error messages)",
+    "description": "The Rust compiler",
+    "repository": "rust-lang/rust",
+    "labels": ["WG-compiler-errors"],
+    "links": [ "about" : "https://paper.dropbox.com/doc/Compiler-errors-FSZdfXAGo3uMQ1wuDcZcy" ],
```

Comment:
These links don't follow [the schemaaaa](https://github.com/nrc/find-work/blob/master/data/schema.md#link)

Reply:
ping @nikomatsakis need to fix this up  -should be `[{"text": "about", "url": "..."}]`